docs/DeveloperGuide.md

```diff
@@ -828,6 +828,27 @@ ___
 * **Mainstream OS**: Windows, Linux, Unix, MacOS
 * **Private contact detail**: A contact detail that is not meant to be shared with others
 
+### Planned Enhancements
+
+1. **Making Phone Number and Email Address Unique**
+
+    The current FriendFolio system uses a contact's name as a unique identifier. We are planning to enhance this by
+    switching to using contact's phone number and email as unique identifiers. This change will prevent multiple
+    individuals from sharing the same email or phone number within the system, while allowing the existence of multiple
+    individuals with the same name.
+
+2. **Improved responsiveness of GUI**
+
+    We are aware that excessively long text, like long names, addresses, and remarks etc. might not display fully in
+    a smaller window. While you are able to make the window larger to display more text, we plan to work on improving the
+    responsiveness of our user interface to handle longer inputs.
```

Comment:
I think can also make mention of this cuz this doesn't really fall into excessively long text ![](https://raw.githubusercontent.com/garywongkai/ped/main/files/c49d1b41-c4bf-4067-978d-3f0f7afab8ba.png)

Reply:
I have also included this in the 3rd point of planned enhancements